src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASKS = 100;
+    public static final String BORDER_DECORATION = "_____________________________________________________";
+    private static Task[] taskLists = new Task[MAX_TASKS];
+
+    public static void displayGreeting() {
+        String logo = " _______     __   __  \n"
+                + "|   _   |[x]| | / / [x]  \n"
+                + "|  | |  || ||  / /  | |\n"
+                + "|  | |  || ||  <    | |\n"
+                + "|__/ |__||_||_| |_| |_|\n";
+        System.out.println("Hello from\n" + logo +"\n" + BORDER_DECORATION);
+        System.out.println("Hello! I'm Niki\nWhat can I do for you?\n" + BORDER_DECORATION);
+    }
+
+    public static void printList() {
+        for (int i = 0; i < Task.getNumberOfTasks(); i++) {
+            System.out.println((i + 1) + "."+ taskLists[i].toString());
+        }
+    }
+
+    public static void addTask(String userInput) {
+        String[] arrayOfTaskStrings = userInput.split(" ");
+        String extractTaskDescription = "";
+        for (int i = 1; i < arrayOfTaskStrings.length; i++) {
+            extractTaskDescription += arrayOfTaskStrings[i] + " ";
+        }
```

Comment:
Perhaps the code logic for extracting the task description from the user input can be moved into a method on its own? This way your `addTask` method will be more focused on the actual adding of the task to the task list.